### PR TITLE
Uncorrelated mutation with multiple steps

### PR DESCRIPTION
**Pull request by JoaoGFarias**
Solves #11 
Dependent on #12 

Details:
- The mutation steps are initiated with the same value, see [comment](https://github.com/miguelarauj1o/ackley/pull/13/files#r64161357)
- The values for specific learning rates are defined via command line arguments, see [comment](https://github.com/miguelarauj1o/ackley/pull/13/files#r64161397)

@miguelarauj1o and @pcstl, I have no idea how to set the initial values of these arguments or if the specific learning rates mutate during execution. Ideas?

### CODE REVIEW

File: ackley/individual/individual.py [hunk 1]
```diff
@@ -60,8 +68,48 @@ def mutate(self):
...
 
         return UncorrelatedMutation(self.args, object_variable, mutation_step)
+
+class UncorrelatedMutationMultipleSteps(BaseIndividual):
+
+    def __init__(self, args, object_variable=None, mutation_step=None):
+        self.chromossome = dict()
+        self.chromossome['mutation_step'] = \
+            [args['mutation_step']] * args['dimension'] \
+            if mutation_step is None else mutation_step
```
Comment (JoaoGFarias) [inline ackley/individual/individual.py]: Definition of mutation steps.
Based on a set passed as argument or a list of copies of the same value, passed via command line arguments

File: ackley/ackley_config.yaml.dist [hunk 2]
```diff
@@ -3,9 +3,11 @@ evolutionary_strategy=intermediary
 mutation_strategy=UncorrelatedMutation
 mutation_step=0.05
 learning_rate=2
+specific_learning_rate=[2,0.5,1,1.5,0.1]
```
Comment (JoaoGFarias) [inline ackley/ackley_config.yaml.dist]: Definition of specific learning rate
